script/zonemaster-cli

```diff
@@ -196,12 +196,14 @@ Default: on
 
 Print all test cases listed in the test modules, then exit.
 
-=item --test=MODULE, --test=MODULE/TESTCASE
+=item --test=MODULE, --test=MODULE/TESTCASE, --test=TESTCASE
 
 Limit the testing suite to run only the specified tests.
 This can be the name of a testing module, in which case all test cases from
 that module will be run, or the name of a module followed by a slash and the
-name of a test case (test case identifier) in that module.
+name of a test case (test case identifier) in that module, or the name of the
+test case.
+Can be specified multiple times.
```

Comment:
> Can be specified multiple times.

I propose that we postpone this clarification until #359. That PR (at least in its current state) makes repeated `--test` arguments overwrite each other. So if we include this PR in the upcoming release and #359 in the next one, we can avoid some flip-flopping in our documentation.